MOBILE_CLIENT/COMMON/SECURE/SECURE_REPUTATION_SCAN/description.md

```diff
@@ -0,0 +1 @@
+Reputation analysis for domain names and IP addresses scanned by multiple APIs and public databases.
```

Comment:
Add more details to say that the reputation score indicate that  it's secure,

Reply:
that info is not accurate, since the issue doesnt always report that the resource is accurate but it can also say that its unreported/unregistered etc...